inc/fields/class-shortcode-ui-field-post-select.php

```diff
@@ -93,6 +93,8 @@ public function action_wp_ajax_shortcode_ui_post_field() {
 		$nonce               = isset( $_GET['nonce'] ) ? sanitize_text_field( $_GET['nonce'] ) : null;
 		$requested_shortcode = isset( $_GET['shortcode'] ) ? sanitize_text_field( $_GET['shortcode'] ) : null;
 		$requested_attr      = isset( $_GET['attr'] ) ? sanitize_text_field( $_GET['attr'] ) : null;
+		$include             = isset( $_GET['include'] ) ? $_GET['include'] : array();
```

Comment:
Just to confirm - does this pass the VIP linter test or does it expect a sanitization function here?